src/config.rs

```diff
@@ -30,7 +33,18 @@ impl Default for UiConfig {
 }
 
 pub fn load() -> anyhow::Result<Config> {
-    let file = std::fs::read_to_string("settings.toml")?;
+    let workspace = PathBuf::from("settings.toml");
+    let xdg_home = ProjectDirs::from("me", "maxjoehnk", "sidenotes")
+        .unwrap()
+        .config_dir()
+        .join("settings.toml");
+    let file: String;
+
+    if workspace.exists() {
+        file = std::fs::read_to_string(workspace)?;
+    } else {
+        file = std::fs::read_to_string(xdg_home)?;
+    }
```

Comment:
FYI
Weil in Rust alles eine Expression is geht das hier auch folgendermaßen:
```suggestion
    let file = if workspace.exists() {
        std::fs::read_to_string(workspace)?
    } else {
        std::fs::read_to_string(xdg_home)?
    };
```

Ist AFAIK auch der weiter verbreitete Weg in Rust sowas abzubilden.

Reply:
Done